minigalaxy/installer.py

```diff
@@ -376,11 +423,9 @@ def _mv(source_dir, target_dir):
 def lang_install(installer: str, language: str):
     languages = []
     arg = ""
-    process = subprocess.Popen(["innoextract", installer, "--list-languages"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    stdout, stderr = process.communicate()
-    output = stdout.decode("utf-8")
+    stdout, stderr, ret_code = _exe_cmd(["innoextract", installer, "--list-languages"])
```

Comment:
This is not a comment about your code, but it is bad that this code will fail silently if there is an error. We should address this at some point, but not in this PR.

Reply:
It shouldnt. I've setup the Default arguments of _exe_cmd to be captureOutput=true and print=False which will lead to both stdout and stderr returned here containing all output.
I can also add a test case to verify that works if you like to

Reply:
I've created an issue for this. I'll close this one.